docs/containerizedcomputation.md

```diff
@@ -0,0 +1,36 @@
+### Start the service:
+- Run `docker compose up funnel`
+- To try it with the rest of the stack, also run:
+`docker compose up htsgetDecrypt`
+
+
+### Access the service:
+- Go into the directory: `cd containerized-computation`
+- Get a token:
+    `token=$(curl -s -k https://localhost:8080/tokens | jq -r '.[0]')`
+- Put it in your task file:
+     `./fixtoken $token task1.json`
```

Comment:
Perhaps this could be simplified by using ´sed´ directly, e.g.:
```
sed -e "s/{{TOKEN}}/$token/g" config/gdi-starter-kit/containerized-computation > task1.json
```